.guides/usage.md

```diff
@@ -0,0 +1,109 @@
+## Key Guidelines
+
+- Always use 2nd-gen functions for new development.
+- Use v1 functions _only_ for Analytics, basic Auth, and Test Lab triggers.
```

Comment:
I thought we covered all of Test Lab triggers:

https://firebase.google.com/docs/functions/test-lab-events